CraftTweaker2-API/src/main/java/crafttweaker/api/tooltip/IngredientTooltips.java

```diff
@@ -47,14 +48,19 @@ public static void addShiftTooltip(IIngredient ingredient, ITooltipFunction func
     }
     
     @ZenMethod
-    public static void clearTooltip(IIngredient ingredient) {
-        CraftTweakerAPI.apply(new ClearTooltipAction(ingredient));
+    public static void clearTooltip(IIngredient ingredient, @Optional boolean leaveName) {
```

Comment:
This is technically breaking on the Java side, it may be better to just have 2 methods overloaded instead of an Optional